City-of-Helsinki/drupal-hdbt-admin
# UHF-X Media library bug fix

# Media library bug fix

## What was done
* Fixed alignment of edit button on media library widget view.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config and hdbt_admin theme
    * `composer require drupal/helfi_platform_config:dev-UHF-X_media_libary_bug_fix`
    * `composer require drupal/hdbt_admin:dev-UHF-X_media_libary_bug_fix`
* Run `make drush-updb drush-cr`

## How to test
* Go to add or edit a node and add image paragraph to the node
* Click on "Add media" and check that the media library links point to correct URLs. F.e. `https://helfi-sote.docker.so/en/health-and-social-services/media/308/edit`
* Check that the layout looks ok - previously the edit icons were indented incorrectly
* Bonus: If you know how to install and use the local proxy (https://github.com/City-of-Helsinki/drupal-helfi-local-proxy), you can try to check the links via local proxy as well. Then the links should point to `https://helfi-proxy.docker.so/fi/sosiaali-ja-terveyspalvelut/media/308/edit`


## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/395
